### ISSUE TITLE
[DOCS] Nicer Readme 

### ISSUE THREAD

**Issue by inson1** [giuspen/cherrytree#2386]
**Is your feature request related to a problem? Please describe.**
I think Readme could look better.

**Describe the solution you'd like**
I would add photo/photos, features, translation lists (we could reuse the lists from your website)

What do you think? Do you have some other ideas?

**Comment by giuspen** [comment]
Yes absolutely, if you would like to propose something with a pull request, that would be great

**Comment by inson1** [comment]
I would add different screenshot tbh. So after merging it isnt still done. If someone has something better, feel free to share it :)
